src/components/listview/listview.js

```diff
@@ -285,8 +285,12 @@ import 'emby-playstatebutton';
 
             if (options.image !== false) {
                 let imgData = options.imageSource === 'channel' ? getChannelImageUrl(item, downloadWidth) : getImageUrl(item, downloadWidth);
-                let imgUrl = imgData.url;
-                let blurhash = imgData.blurhash;
+                let imgUrl;
```

Comment:
Do note that the blurhash changes to ListView were removed late in 10.6 (after the `es6` branch was spun off) due to visual issues.

Reply:
Let's avoid merging this one for now then. I've been trying to avoid any ES6 migrations that might make the final merge more difficult.

Reply:
Agreed

Reply:
I had to add the if just below because in my library there are some songs without album art and it completely broke list view and didn't render any of it. Do you want me to target master instead to make it easier to merge?

Reply:
It's targeting master now, so no changes to listview are required